docs/src/modules/utils/useCodeCopy.ts

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import copy from 'clipboard-copy';
+
+export const setupKeyboardCopy = (copyBtn: HTMLButtonElement) => {
+  if (!copyBtn) {
+    return;
+  }
+  copyBtn.addEventListener('keydown', (event) => {
+    if (event.code === 'KeyC' && (!!event.metaKey || !!event.ctrlKey) && !event.shiftKey) {
+      copyBtn.click();
+      if (typeof window !== 'undefined' && 'ga' in window) {
+        // @ts-ignore
+        window.ga('send', {
+          hitType: 'event',
+          eventCategory: copyBtn.getAttribute('data-ga-event-category') || 'code',
+          eventAction:
+            // the button's `data-ga-event-action` usually is `copy-click`
+            copyBtn.getAttribute('data-ga-event-action')?.replace('click', 'keyboard') ||
+            'copy-keyboard',
+          eventLabel: copyBtn.getAttribute('data-ga-event-label'),
+        });
+      }
+    }
+  });
+  copyBtn.parentElement?.addEventListener('mouseenter', () => {
+    copyBtn.focus();
```

Comment:
This hijacks page scroll with Space key if cursor hovers a code block:

https://user-images.githubusercontent.com/13808724/164423454-010512a8-42f0-47be-8d1a-0c494010ce43.mov

Do we really need to focus copy button on hover?

Reply:
I see. I will try another approach.

Reply:
Done! please try again, no more focus.

Reply:
Awesome!